### DISCUSSION TITLE
[05_Age_Gender_Detection]Is it possible to process the image of the camera input?

### DISCUSSION THREAD

**Discussion by maho-sk** [Ignitarium-Renesas/RZV2L_AiLibrary#4] [General]
I have questions regarding”05_Age_Gender_Detection”.

・I understand that the current library inputs and processes image data. Will you release a version of the library with images input from the camera?

・If the camera input version is not released, is it possible to improve it to the camera input version based on the current library (image data input)?

Will the development man-hours be reduced by modifying the current library rather than developing everything from scratch by the user?

**Comment by mohitb-ign** [comment]
Yes, a version supporting video feed from camera is under progress. It will be released soon. Reduction of man-hours depend on multiple factors like desired accuracy and desired speed. 
The provided DL model is a simple model and works well for prototype development. The same or similar model architectures can be quickly integrated and improved for production grade applications
